data_structures/route_parameters.cpp

```diff
@@ -117,6 +117,15 @@ void RouteParameters::addTimestamp(const unsigned timestamp)
     }
 }
 
+void RouteParameters::addBearing(const int bearing)
+{
+    bearings.resize(coordinates.size());
+    if (!bearings.empty())
+    {
+        bearings.back() = bearing;
+    }
+}
```

Comment:
Correct, I've just mimicked the behaviour of `timestamp`.  In theory we could supply bearings only on some coordinates, but I didn't dig that far into the logic for now.